lib/cn/kvset_builder_internal.h

```diff
@@ -68,6 +74,7 @@ struct kvset_builder {
     struct kmd_info sec;
```

Comment:
I think changing names of `main` and `sec` members to `kblk_kmd` and `hblk_kmd` would be improve readability.

Reply:
I think this makes a lot of sense.